shared/src/ledger/native_vp/masp.rs

```diff
@@ -116,6 +116,7 @@ where
         keys_changed: &[&Key],
         transaction: &Transaction,
     ) -> Result<bool> {
+        println!("\nCHECKING VALID NULLIFIERS REVEAL\n");
```

Comment:
Just make sure to remove these prints before merging

Reply:
done in a1143ab7f88fa58cce305c6981dfdb6924887476